app/src/components/common/layout/header/index.tsx

```diff
@@ -284,6 +288,15 @@ const HeaderContainer: React.FC = (props: any) => {
             />
           )}
 
+          <HeaderButton>
+            {relay
+              ? `${formattedxOmenBalance.slice(0, formattedOmenBalance.length - 4)}`
+              : `${formattedOmenBalance.slice(0, formattedOmenBalance.length - 4)}`}
```

Comment:
I would suggest that it's maybe better to change formatting of Omen token in useConnectedBalance since for xOmen we have 2 decimals and for Omen 3 decimals it causes this on xDai...Maybe we could change it in useConnectedBalance to always have zero decimals so we don't have to format it anywhere else @pimato ?
![Screenshot 2021-05-03 at 12 34 11](https://user-images.githubusercontent.com/33226956/116866977-84a62c80-ac0c-11eb-9ef0-7ce47b658d85.png)